boards/demo_f030c8.json

```diff
@@ -0,0 +1,38 @@
+{
+  "build": {
+    "cpu": "cortex-m0",
+    "extra_flags": "-DSTM32F030x8",
+    "f_cpu": "48000000L",
+    "mcu": "stm32f030c8t6",
+    "variant": "DEMO_F030C8"
+  },
+  "debug": {
+    "default_tools": [
+      "stlink"
+    ],
+    "jlink_device": "STM32F030C8",
+    "onboard_tools": [
+      "stlink"

```

Comment:
Are you sure there is an ST-Link on board?

Reply:
No, It has not. so remove it.